.github/workflows/e2e-test-tf-module.yml

```diff
@@ -112,6 +111,37 @@ jobs:
           EOF
           cat terraform.tfvars
 
+      - name: Create GCP Terraform variable input file
+        if: inputs.cloudProvider == 'gcp'
+        working-directory: ./terraform-module/gcp-constellation
+        run: |
+          cat > terraform.tfvars <<EOF
+          name = "${{ steps.create-prefix.outputs.prefix }}"
+          project = "${{ secrets.GCP_E2E_PROJECT }}"
+          service_account_id = "${{ steps.create-prefix.outputs.prefix }}-sa"
+          image = "${{ inputs.image }}"
+          zone = "${{ inputs.regionZone || 'europe-west3-b' }}"
```

Comment:
I think we should rather add a default value in the `inputs` section, it doesn't need duplication there and is more accessible.

Reply:
Leo wanted exactly this since the default is provider specific and we don't want a region input for each csp.